features/promotion/prevent_deletion_of_promotion_in_use.feature

```diff
@@ -0,0 +1,21 @@
+@promotion
+Feature: Prevent deletion of promotions which are in use
+    In order to maintain proper payment history
+    As an Administrator
+    I want to be prevented from deleting promotions which are in use
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And the store has a product "PHP Mug"
+        And there is a promotion "Holiday promotion"
+        And the promotion gives "10%" percentage discount on shipping to every order
+        And the customer "john.doe@gmail.com" placed an order "#00000022"
+        And the customer chose "Free" shipping method to "France" with "Cash on Delivery" payment
```

Comment:
Free shipping with 10% discount does not make much sense, perhaps we should use flat rate here.